packages/salmon-wallet/src/SalmonWalletMnemonic.ts

```diff
@@ -0,0 +1,36 @@
+import { Network } from '@defichain/jellyfish-network'
+import { JellyfishWallet, WalletAccountProvider } from '@defichain/jellyfish-wallet'
+import { Bip32Options, MnemonicHdNode, MnemonicHdNodeProvider } from '@defichain/jellyfish-wallet-mnemonic'
+import { WhaleWalletAccount } from '@defichain/whale-api-wallet'
+
+const PURPOSE_LIGHT_ORACLE_ADAPTER = 2
+
+/**
+ * Basic wrapper around JellyfishWallet, so we can extend with any oracle specific
+ * functionality we might need
+ */
+
+export class SalmonWalletMnemonic extends JellyfishWallet<WhaleWalletAccount, MnemonicHdNode> {
```

Comment:
```suggestion
/**
 * Basic wrapper around JellyfishWallet, so we can extend with any oracle specific
 * functionality we might need
 */
export class SalmonWalletMnemonic extends JellyfishWallet<WhaleWalletAccount, MnemonicHdNode> {
```

JSDoc should be better written, it's too generic?